library/webpack-plugins/react-universal-plugin.js

```diff
@@ -147,11 +147,9 @@ function createWebCompiler(compiler, getEntries) {
     normalModuleFactory.plugin('after-resolve', (data, done) => {
       const { loaders, rawRequest, resourceResolveData: { query } } = data
 
-      if (query === '?universal-client')
-        loaders.push({ loader: require.resolve('../webpack-loaders/react-universal-client-loader') })
+      if (query === '?universal-client') { loaders.push({ loader: require.resolve('../webpack-loaders/react-universal-client-loader') }) }
```

Comment:
Here I don't like the extra noise of the `{` and `}` and on top of that we should probably adjust our line length as well. I rather have that statement under the `if`, indented as it was.